beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -220,6 +222,11 @@ def post_processing_fn(inputs):
 
         return post_processing_fn
 
+    @staticmethod
+    def _min_adj(bits, low, range_len, narrow_range):
+        return range_len / (2 ** bits - (2 if narrow_range else 1)) * \
+            tf.round((2 ** bits - (2 if narrow_range else 1)) * low / range_len)
```

Comment:
Could you refactor the equation to make the code more readable?

Reply:
Please check